Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,9 @@
 import React from 'react';
 import './Card.css';
+
+
+const CardBanner = () => {
+    return <img src='https://tk-assets.lambdaschool.com/fcd75197-7d12-46ec-bc9e-4130f34822fa_reactbackground.png' alt=''></img>
```

Comment:
The image tag is a self-closing tag, look into it and would like to see you implement it :)